index.html

```diff
@@ -8,195 +8,228 @@
-            <section class="head-body-container">
-                <div class="head-body">
-                    <h1>I’m Echezona <br> Glad to see you!</h1>
-                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-                    <h3>LET'S CONNECT</h3>
-                    <ul>
-                        <li><a href=""><img src="image/Vector.png" alt=""></a></li>
-                        <li><a href=""><img src="image/Vector(1).png" alt=""><img src="image/Vector(2).png" alt=""></a></li>
-                        <li><a href=""><img src="image/Vector(3).png" alt=""></a></li>
-                        <li><a href=""><img src="image/Vector(4).png" alt=""></a></li>
-                        <li><a href=""><img src="image/Vector(5).png" alt=""></a></li>
-                    </ul>
-                </div>
-            </section>
+  <section id="head">
+      <nav>
+        <div class="nav-inner">
+          <div class="logo"><a href="">ECHEZONA</a></div>
+          <a href="#" class="toggle-button">
+            <svg class="bar" width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="#6070FF"/>
+            </svg>
+          </a>
+          <ul id="hamburger-menu">
+            <li class="nav-links"><a href="#portfolio">Portfolio</a></li>
+            <li class="nav-links"><a href="#about">About</a></li>
+            <li class="nav-links"><a href="#contact">Contact</a></li>
+          </ul>
         </div>
-    </div>
+      </nav>
+          
+      <div class="background-image">
+        <div class="content">
+          <h1>I’m Echezona <br> Glad to see <span>you!</span></h1>
+          <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <h3>LET'S CONNECT</h3>
+          <ul>
+            <li><a href=""><img src="image/Vector.png" alt=""></a></li>
+            <li><a href=""><img src="image/Vector(1).png" alt=""><img src="image/Vector(2).png" alt=""></a></li>
+            <li><a href=""><img src="image/Vector(3).png" alt=""></a></li>
+            <li><a href=""><img src="image/Vector(4).png" alt=""></a></li>
+            <li><a href=""><img src="image/Vector(5).png" alt=""></a></li>
+          </ul>
+        </div>
+      </div>
+  </section>
```

Comment:
There is a border-radius on the bottom-left corner of the headline section which is missing in your design, also there's suppose to be a space between the headline section and that of the first card component which is missing, kindly check and fix these anomalies.

### Your design 

![Screenshot from 2023-01-27 19-14-28](https://user-images.githubusercontent.com/65233860/215178996-f0b9be9e-8062-47d4-80e3-5e9b99b0b8f2.png)


### Figma design 

![Screenshot from 2023-01-27 19-19-09](https://user-images.githubusercontent.com/65233860/215179058-bb137916-c18b-4ddb-a216-da953ef17822.png)